src/fedora/33/amd64/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM fedora:33
```

Comment:
Does the CI verify fedora:33 is a good enough base for this queue❔ If, for example, you copied this docker file from one for Fedora 32, are the two versions different in some way that might be a concern❔

Reply:
yes; if all the commands executed don't succeed, this CI will fail.  I also built both images locally and checked them out before signing off.

Reply:
Cool, so after you guys merge this PR, how will I know the exact docker string for the helix queue to try in our builds?

Reply:
It will get automatically recorded in https://github.com/dotnet/versions/blob/master/build-info/docker/image-info.dotnet-dotnet-buildtools-prereqs-docker-master.json